app/src/main/java/com/github/campus_capture/bootcamp/fragments/ScoreboardFragment.java

```diff
@@ -58,7 +58,10 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
         RecyclerView recyclerView = view.findViewById(R.id.scoreboard_view);
         recyclerView.setLayoutManager(new LinearLayoutManager(getContext()));
 
-        recyclerView.setAdapter(new ScoreRecyclerViewAdapter(firebaseInterface.getScores()));
+        // TODO not sure how to refactor this correctly using the new future interface
+        try {
+            recyclerView.setAdapter(new ScoreRecyclerViewAdapter(backendInterface.getScores().get()));
```

Comment:
Maybe add a toast if the operation fails... and some refresh button

Reply:
I'll leave that to a futur UI task I think